### PR TITLE
restructured to allow for multiple jobs, locations, distances. Added …

### PR DESCRIPTION
**Pull request by Developer-Bear**
…notification instead of esx_phone use.

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -23,24 +27,35 @@ end)
...
+		end
     end
 end)
+
+RegisterNetEvent('esx_motiondetector:Notify')
+AddEventHandler('esx_motiondetector:Notify', function(coords)
+	if Config.NotificationType == 1 then
+		TriggerServerEvent('esx_phone:send', Config.Job, 'An intruder is within ' ..Config.Distance.. 'm of your hideout ', true, {x = coords.x, y = coords.y, z = coords.z})
+	elseif Config.NotificationType == 2 then
+		ESX.ShowNotification('An intruder is within ' ..Config.Distance.. 'm of your hideout ', false, true)
```
Comment (BrutalBirdie) [inline client/main.lua]: `Config.Distance` will always result in an error since you deleted `Config.Distance` in the `config.lua`

Comment (Developer-Bear) [reply client/main.lua]: Yeah just from the notifications though. I changed up the notification to resolve this. Thanks!

**Comment by mortdudley** [comment]
Apologies for taking so long to go through your PR, Unfortunately, our server has been taking up way too much of our time lately. I'll go through the PR now.

**Comment by Developer-Bear** [comment]
> Apologies for taking so long to go through your PR, Unfortunately, our server has been taking up way too much of our time lately. I'll go through the PR now.

No worries :)

**Pull request merged**